src/components/task-plan/reading.cjsx

```diff
@@ -82,91 +99,99 @@ SelectTopics = React.createClass
       </div>
     </BS.Modal>
 
+ReadingFooter = React.createClass
+  render: ->
+    classes = []
+    classes.push('disabled') unless @props.enabled
```

Comment:
Is `enabled` defined in `propTypes`? Maybe this concern should be a separate PR. Or, this logic should be inside the footer instead of as a Prop.